Tests/PointFreeTests/EpisodeTests.swift

```diff
@@ -8,25 +8,48 @@ import XCTest
 @testable import HttpPipeline
 import HttpPipelineTestSupport
 import Optics
+#if !os(Linux)
+  import WebKit
+#endif
 
 class EpisodeTests: TestCase {
-//  func testHome() {
-//    let request = URLRequest(url: URL(string: "https://localhost:8080/episodes/ep6-the-algebra-of-predicates-and-sorting-functions")!)
-//      |> \.allHTTPHeaderFields .~ [
-//        "Authorization": "Basic " + Data("hello:world".utf8).base64EncodedString()
-//    ]
-//
-//    let conn = connection(from: request)
-//    let result = conn |> siteMiddleware
-//
-//    assertSnapshot(matching: result.perform(), record: true)
-//    assertWebPageSnapshot(matching: result.perform(), record: true)
-//  }
-
-  func testEpisodeHtml() {
-    assertSnapshot(
-      matching: prettyPrint(nodes: episodeView.view(episodes.last!), pageWidth: 220),
-      pathExtension: "html"
-    )
+  func testEpisodePage() {
+    let request = URLRequest(url: URL(string: url(to: .episode(.left(episodes.first!.slug))))!)
+      |> \.allHTTPHeaderFields .~ [
+        "Authorization": "Basic " + Data("hello:world".utf8).base64EncodedString()
+    ]
+
+    let conn = connection(from: request)
+    let result = conn |> siteMiddleware
+
+    assertSnapshot(matching: result.perform())
+
+    #if !os(Linux)
+      if #available(OSX 10.13, *) {
```

Comment:
we have to double `if` this :/ once for linux, and once for macOS API check. once circleci upgrades to 10.13 we can at least remove one of em